### PR TITLE
WIP on ADR 033 x/bank

### PR DESCRIPTION
**Pull request by aaronc**

### CODE REVIEW

File: types/module/server/manager.go [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+		// check if we actually have a server module, otherwise skip
+		serverMod, ok := mod.(Module)
+		if !ok {
+			continue
+		}
+
+		serverMod.RegisterInterfaces(mm.cdc.InterfaceRegistry())
+	}
+
+	for _, mod := range modules {
```
Comment (jgimeno) [inline types/module/server/manager.go:47]: you can use the same loop as before I think :)

Comment (aaronc) [reply types/module/server/manager.go:47]: I want to make sure all interface types are registered first

**Comment by clevinson** [comment]
@aaronc is this something that should be closed? Or are you planning to continue working on this?

**Comment by aaronc** [comment]
We can close. Future work will happen in the SDK when we have enough dev bandwidth

**Pull request closed without merging**